src/components/Deployment/Deployment.tsx

```diff
@@ -185,48 +176,65 @@ const Deployment = () => {
           ))}
         </Body>
       </Wrapper>
-    </Card>
+    </MenuModalWrapper>
   );
 };
 
 export default Deployment;
 
 const Wrapper = styled.div`
-  margin-top: 10px;
+  margin: 12px 0px;
+`;
+
+const ConfirmMessageWrapper = styled.p`
+  padding: 16px 0px;
+  text-align: center;
+`;
+
+const ConfirmDeployButton = styled.div`
+  display: flex;
+  justify-content: center;
+  background: ${({ theme }) => theme.color.background.settingMenuMain};
+  color: #fff;
```

Comment:
Are we sure it's always gonna be white? We're not gonna let the user decide color here?